src/web/src/components/SearchResults.tsx

```diff
@@ -54,7 +54,9 @@ const SearchResults = () => {
   const { textParam, filter } = useSearchValue();
 
   const prepareUrl = (index: number) =>
-    `${telescopeUrl}/query?text=${encodeURIComponent(textParam)}&filter=${filter}&page=${index}`;
+    `${searchServiceUrl}/query?text=${encodeURIComponent(
```

Comment:
it works for me without `query` in the URL

```suggestion
    `${searchServiceUrl}?text=${encodeURIComponent(
```

Reply:
If you want to keep using `query`, [this line](https://github.com/Seneca-CDOT/telescope/blob/bf293f64cc7919c832f4f57326e107f916951660/src/api/search/src/index.js#L7) should be:

```js
service.router.use('/query', query);
```

Reply:
We don't want `/query` any more, please remove.